.travis.yml

```diff
@@ -24,6 +24,7 @@ script:
   - ls -la bin/linux/amd64
   - amp --server=localhost cluster create --tag=local --registration=none --notifications=false
   - TESTINCLUDE=platform/testing testrunner platform/tests
+  - tests/run
```

Comment:
We shouldn't need to build an image for the CI build. The entire environment is already set up, so you should be able to execute `go test -v github.com/appcelerator/amp/tests/integration/` directly (or a script that executes that directly. Building the image adds a *ton* of time to the CI build, although it makes sense for local testing (non-CI testing).